mopidy/core/library.py

```diff
@@ -136,9 +139,16 @@ def get_distinct(self, field, query=None):
         validation.check_choice(field, validation.DISTINCT_FIELDS)
         query is None or validation.check_query(query)  # TODO: normalize?
 
+        compat_field = {"track_name": "track"}.get(field, field)
+        if compat_field is not field:
```

Comment:
I think we want just regular equality here, not instance check.

```suggestion
        if compat_field != field:
```